pallets/messages/src/lib.rs

```diff
@@ -120,17 +121,6 @@ pub mod pallet {
 	#[pallet::without_storage_info]
 	pub struct Pallet<T>(_);
 
-	/// A temporary storage of messages, given a schema id, for a duration of block period.
```

Comment:
this is removed after some local testing, message add worked and the old storage did not appear after upgrade as expected!

![Screen Shot 2022-12-14 at 3 39 14 PM](https://user-images.githubusercontent.com/9152501/207739062-61d2a195-6b87-4312-be36-cadbcd3a134b.png)
![Screen Shot 2022-12-14 at 3 39 33 PM](https://user-images.githubusercontent.com/9152501/207739069-022c2e66-6530-443b-9f0c-2a22781721a0.png)